server/events/command/context.go

```diff
@@ -18,6 +18,9 @@ const (
 
 	// Commands that are triggered by comments (ie. atlantis plan)
 	CommentTrigger
+
+	// Commands that are triggered by PR reviews (ie. approving a PR)
+	PRReviewTrigger
```

Comment:
nit: do we have a conditional that checks on `PRReviewTrigger` similar to [AutoTrigger](https://github.com/lyft/atlantis/blob/1f787e85e12af8506299bda2d6a7c27cd2ede6d7/server/events/plan_command_runner.go#L174)? If not, maybe we can use `AutoTrigger` since PRReviews are triggered automatically?

Reply:
Sure thing, changed it.